packages/wrangler/src/user/env-vars.ts

```diff
@@ -1,12 +1,41 @@
 import { getEnvironmentVariableFactory } from "../environment-variables";
 
-/**
- * Try to read the API token from the environment.
- */
-export const getCloudflareAPITokenFromEnv = getEnvironmentVariableFactory({
+export type ApiCredentials =
+  | {
+      apiToken: string;
+    }
+  | {
+      authKey: string;
+      authEmail: string;
+    };
+
+const getCloudflareAPITokenFromEnv = getEnvironmentVariableFactory({
   variableName: "CLOUDFLARE_API_TOKEN",
   deprecatedName: "CF_API_TOKEN",
 });
+const getCloudflareGlobalAuthKeyFromEnv = getEnvironmentVariableFactory({
+  variableName: "CLOUDFLARE_GLOBAL_API_KEY",
```

Comment:
```suggestion
  variableName: "CLOUDFLARE_API_KEY",
```
we can remove this GLOBAL prefix everywhere!